src/Middleware/NodeServices/src/Microsoft.AspNetCore.NodeServices.csproj

```diff
@@ -2,6 +2,7 @@
   <PropertyGroup>
     <Description>Invoke Node.js modules at runtime in ASP.NET Core applications.</Description>
     <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
+    <IsShipping>false</IsShipping>
```

Comment:
@dougbu Is this enough to make sure the package doesn't get published?

FYI this has already been discussed offline (as per the related announcement)

Reply:
We'll do further cleanup in RC2/RTM to remove it completely from 5.0

Reply:
@javiercn this is normally used to create a non-shipping package another repo needs. Suggest `<IsPackable>false</IsPackable>` to get rid of the package entirely.

Reply:
Added that.